ChangeLog.md

```diff
@@ -524,7 +529,7 @@ See docs/process.md for more on how version tagging works.
   (#18861)
 - The `emscripten_proxy_async_with_callback` API was replaced with a simpler
   `emscripten_proxy_callback` API that takes a second callback to be called if
-  the worker thread dies before completing the proxied work.  
+  the worker thread dies before completing the proxied work.
```

Comment:
Can you revert all these unralated changes to the ChangeLog where newlines are removed?

BTW, the way I setup my editor is to (1) show trailing whitespace and (2) have a shortcut to strip it all, but not to have it do this automatically.. otherwise you end up with changes like this.

Reply:
Thanks for the insights and sorry for the inconvenience.